hooks/useEasyWrite.ts

```diff
@@ -55,8 +57,14 @@ export const useEasyWrite = (
     );
     if (!config.request) return;
     // succeed
-    if (hash && transactionData?.status === 'success' && prev.status !== 'success') {
+    if (
+      hash &&
+      transactionData?.status === 'success' &&
+      prev.status !== 'success' &&
+      !prev.isNotificationShownAlready
+    ) {
       console.log('notify success');
+      setIsNotificationShownAlready(true);
```

Comment:
This works for the first bridge, but if I try to bridge twice than my success toast doesn't get shown. We have to reset the `isNotificationShownAlready` state when a new toast is fired.

Reply:
Fixed here: [7326eda](https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/146/commits/7326edab7fe672b9a62e1e3e34dd93439db1428b)

Reply:
Do you think the overall approach of this PR is good and can be taken out of Draft?

Reply:
Good idea, like this? https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/146/files

Reply:
Yes, exactly!

Reply:
Awesome, thanks! will take out of draft now.